astropy/table/tests/conftest.py

```diff
@@ -69,7 +69,7 @@ class MyTable(table.Table):
 # and masked (MaskedArray) column.
 
 
-@pytest.fixture(params=['unmasked', 'masked', 'subclass'])
+@pytest.fixture(params=['unmasked', 'masked', 'subclass', 'quantity'])
```

Comment:
This adds a new case to a lot (hundreds?) of tests since this fixture is used a lot. While on principle it never hurts to add more tests, I'm not sure if this is global addition is helping our coverage or just burning CI time and carbon.

Reply:
Yes, initially I was only looking for a simple way to add `QTable` coverage to some tests...
This is adding 359 tests to the existing 2079 tests in `table` – I found about 0.8 s or 5 % longer runtime for that.
Don't know if that is worth it, or `QTable` is just not that critical compared to `MaskedTable` or the subclass.